src/Chapter1.hs

```diff
@@ -653,7 +666,11 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n
+  | div (abs n) 10 == 0 = abs n
+  | div (abs n) 10 > 0 = firstDigit (div (abs n) 10)
```

Comment:
That is right 💯 

You can see that `abs n` is reused several times in this function. This usually means that you can move it into the `where` case and reuse everywhere 👍🏼